pkg/controllers/disruption/suite_test.go

```diff
@@ -1833,18 +1833,20 @@ var _ = Describe("Metrics", func() {
 	})
 	It("should fire metrics for single node empty disruption", func() {
 		nodeClaim, node := nodeClaims[0], nodes[0]
```

Comment:
Do we not need to have this node marked as consolidatable for this metric to fire? If not, do we have a bug?